linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/rule.yml

```diff
@@ -41,6 +41,14 @@ rationale: |-
     utilize authentication that meets industry and government requirements. For government systems, this allows
     Security Levels 1, 2, 3, or 4 for use on {{{ full_name }}}.
 
+{{% if product in ["fedora", "rhel8","ol8"] %}}
+warnings:
+    - general: |-
+        This rule seems to be included in a benchmark of a platform that supports crypto policies.
+        Crypto policies centralise crypto algorithms configuration across the whole system, and unless overriden in a very specific way, they take priority over legacy configuration, such as one described by this rule.
+        Therefore, this rule is irrelevant and shouldn't even be part of the {{{ full_name }}} benchmark at all.
+{{% endif %}}
```

Comment:
Instead, this rule should specify `prodtype` followed by a list of product that do not have system-wide crypto policies. Then, the rule would never get included into benchmarks in which it doesn't make sense, therefore you won't need to add this warning there.